src/aura/RateTrial/RateTrialHelper.js

```diff
@@ -0,0 +1,30 @@
+({
+
+    /**
+        *   @author: Dragomir Rangelov
+            @date: 11-10-2018
+            @description: Queue up a remote method call to saveRating Apex controller to create a record
+            @param:
+            @returns:
+        **/
+
+    createRating: function(component, newRating) {
+        var action = component.get("c.saveRating");
+
+        action.setParams({
+            "rating": newRating
+        });
+        action.setCallback(this, function(response) {
+            var state = response.getState();
+
+            if (state === "SUCCESS") {
+
+                alert("Your rating has been submitted!");
```

Comment:
Not needed as alert is not styled - mismatch in user experience. If you want to have confirmation for a user use popup with lightning styling.